### DISCUSSION TITLE
Unit tests: Mock certain grains

### DISCUSSION THREAD

**Discussion by ekrak** [dotnet/orleans#8543] [Q&A]
Hi all,

I'm using TestCluster for unit testing and based on the available documentation it works fine.
My question is what to do with the dependant grains and if there is a possibility to mock them somehow.

i.e.: Grain1 calls Grain2
We are testing Grain1 and we want to mock Grain2. I do not see a way to inject this somehow into GrainFactory.

Any way to achieve this?

**Comment by Schwobaland** [comment]
Same question as #7502 :)
